procedures: starting a new workspace (#2227)
codenvy/che-docs: modules/user-guide/pages/url-of-your-che-dashboard.adoc

:_content-type: CONCEPT
:description: URL of your {prod-short} dashboard
:keywords: dashboard, manage-workspaces, verify, workspace-status, workspace-state
:navtitle: URL of your {prod-short} dashboard
// :page-aliases:

[id="url-of-your-{prod-id-short}-dashboard_{context}"]
= URL of your {prod-short} dashboard

> The *Workspaces* page of your {prod-short} dashboard is where you manage your workspaces and verify their current states. The URL syntax for this page is like so:

Reviewer: ```suggestion
The *Workspaces* page of your {prod-short} dashboard is where you manage your workspaces and verify their current states. The URL syntax for this page is as follows:
```
